### PR TITLE
Fix body handler

### PR DESCRIPTION
**Pull request by dstepanov**

### CODE REVIEW

File: test-suite-kotlin-ksp/src/test/kotlin/io/micronaut/annotation/processing/SuspendMethodSpec.kt [hunk 1]
```diff
@@ -37,7 +37,7 @@ class SuspendMethodSpec {
         val res = Flux.from(client
             .retrieve(
                 HttpRequest.GET<Any>("/sync/any"),
-                Any::class.java
+                String::class.java
```
Comment (dstepanov) [inline test-suite-kotlin-ksp/src/test/kotlin/io/micronaut/annotation/processing/SuspendMethodSpec.kt:40]: Not sure why this worked before

**Review by yawkat — APPROVED** [review]

File: test-suite-kotlin-ksp/src/test/kotlin/io/micronaut/annotation/processing/SuspendMethodSpec.kt (hunk 1 above)
Comment (dstepanov) [reply test-suite-kotlin-ksp/src/test/kotlin/io/micronaut/annotation/processing/SuspendMethodSpec.kt:40]: Fixed

**Comment by dstepanov** [comment]
Had to add `TextPlainObjectBodyReader` for special cases like this

**Comment by yawkat** [comment]
fine

**Pull request merged**